### ISSUE TITLE
BUG: difficult to sign out via VoiceOver on iOS in some situations

### ISSUE THREAD

**Issue by eli-oat** [department-of-veterans-affairs/va-mobile-app#12503]
**NOTE** Because we are not currently able to reproduce this behavior, this ticket exists for tracking purposes while we investigate further.

The initial conversation around this started in [DSVA Slack](https://dsva.slack.com/archives/C02M11AKKNU/p1770048135959689).

***

via @artsymartha68: 

Some weirdness with the Sign Out option and VoiceOver (VO) on reported on iPhone and iPad. 

- iPhone iOS 26.1.2 
- Log out: couldn’t grab the log out button in the app with VO
- Red sign out bar is very small vertically with VO
- Had to turn VO off to get, then it was bigger and it read sign out
- Flicking left to right wouldn’t grab it
- On iPad - very slim vertically too
